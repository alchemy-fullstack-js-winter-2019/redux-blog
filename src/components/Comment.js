import React from 'react';
import PropTypes from 'prop-types';

function Comment({ comment }) {
  return (
    <>
      <p>{comment.body}</p>
      <h4>-{comment.email}</h4>
    </>
  );
}

Comment.propTypes = {
  comment: PropTypes.object.isRequired
};

export default Comment;
