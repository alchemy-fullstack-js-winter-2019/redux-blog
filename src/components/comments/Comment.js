import React from 'react';
import PropTypes from 'prop-types';
import styles1 from '../app/App.css';

export default function Comment({ comment }) {
  return (
    <div className={styles1.comment}>
      <p>{comment.body}</p>
      <p>Email: {comment.email}</p>
    </div>
  );
}

Comment.propTypes = {
  comment: PropTypes.object.isRequired
};
