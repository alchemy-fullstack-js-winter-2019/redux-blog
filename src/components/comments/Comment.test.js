import React from 'react';
import renderer from 'react-test-renderer';
import Comment  from './Comment';

describe('Comment', () => {
  it('matches a snapshot', () => {
    const comment = {
      body: 'comment',
      email: 'email@email.com'
    };
    const tree = renderer.create(
      <Comment comment={comment}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
