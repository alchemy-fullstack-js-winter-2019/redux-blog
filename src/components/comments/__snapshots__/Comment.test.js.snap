// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment matches a snapshot 1`] = `
Array [
  <p>
    comment
  </p>,
  <p>
    Email: 
    email@email.com
  </p>,
]
`;
